docs/buckets.rst

```diff
@@ -0,0 +1,62 @@
+Buckets
+#######
+
+Buckets are a way to group users around some data.
+
+Basically has got an and a list of principals that can administrate
+it.
+
+A bucket can be created using a PUT on the bucket URI:
+
+.. code-block:: http
+
+   > PUT /buckets/servicedenuages HTTP/1.1
+   < 201 Created
+
+    {
+      "id": "servicedenuages",
+      "principals": ["email:natim@example.com"]
+    }
+
+
+There are two kinds of data linked to a bucket:
+
+ - collections
+ - groups
```

Comment:
I don't understand what a role is, but I do with a group. Why are you suggesting we could use "role"?